docs/oscar_references.bib

```diff
@@ -1467,8 +1478,9 @@ @Misc{OEIS
   bibkey        = {OEIS},
   author        = {OEIS Foundation Inc.},
```

Comment:
Does it help to `{}`-protect the entry? It would in real BibTeX, probably not here, but worth a try?
```suggestion
  author        = {{OEIS Foundation Inc.}},
```